src/install/db.py

```diff
@@ -26,31 +14,16 @@ def _get_db_directory():
     return output.strip()
 
 
-def _add_mongo_mirror(distribution):
-    apt_key_output, apt_key_code = execute_shell_command_get_return_code(
-        MONGO_MIRROR_COMMANDS[distribution]['key']
-    )
-    tee_output, tee_code = execute_shell_command_get_return_code(
-        MONGO_MIRROR_COMMANDS[distribution]['sources']
-    )
-    if any(code != 0 for code in (apt_key_code, tee_code)):
-        raise InstallationError('Unable to set up mongodb installation\n{}'.format('\n'.join((apt_key_output, tee_output))))
-
-
-def main(distribution):
+def main():
     logging.info('Setting up mongo database')
-
-    if distribution in ['xenial', 'debian']:
-        _add_mongo_mirror(distribution)
-        apt_update_sources()
-        apt_install_packages('mongodb-org')
-    else:
-        apt_install_packages('mongodb')
+    _, return_code = execute_shell_command_get_return_code('docker pull mongo:3-xenial')
```

Comment:
Why xenial with EOL next year?

Reply:
sadly newer versions like bionic are only available with MongoDB 4
I tried to build a custom container but was not able to connect to it (for still unknown reasons)

Reply:
replaced with a docker image based on debian stretch